### PR TITLE
Added a bridging use case + an in-depth article

### PR DESCRIPTION
**Pull request by shelleyolivia**
Hi there, two changes:

1) Added a concrete bridging use case - under the "why do we need bridging" section, I added in a concrete example to solidify the point of what bridging looks like in company that helps bridges Ethereum to other chains. The thinking is that the additional detail adds value to understanding why multi chain support and bridging is important

2) Added an in-depth article that has been recently updated - the other docs were all from 2021

Pls lmk if there are any questions, thanks!

### CODE REVIEW

File: src/content/bridges/index.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ You're from the USA and are planning a trip to Europe. You have USD, but you nee
 
 But, what do you do if you want to make a similar exchange to use a different blockchain? Let's say you want to exchange ETH on Ethereum Mainnet for ETH on [Arbitrum](https://arbitrum.io/). Like the currency exchange we made for euros, we need a mechanism to move our ETH from Ethereum to Arbitrum. Bridges make such a transaction possible. In this case, [Arbitrum has a native bridge](https://bridge.arbitrum.io/) that can transfer ETH from Mainnet onto Arbitrum.
 
+
```
Comment (minimalsm) [inline src/content/bridges/index.md:24]: Extra space

```suggestion
```

File: src/content/bridges/index.md [hunk 2]
```diff
@@ -36,6 +38,8 @@ Bridges enable:
 - users to access new platforms and leverage the benefits of different chains.
 - developers from different blockchain ecosystems to collaborate and build new platforms for the users.
 
+For instance, for a company like Socket, [bridging is essential](https://www.alchemy.com/blog/alchemy-powering-socket-the-bridging-aggregator) to building a multi-chain ecosystem and faciliating value exchange between L1s like Ethereum to other L1 and L2 protocols.
+
```
Comment (minimalsm) [inline src/content/bridges/index.md:42]: I don't think this adding this to this section adds any value. I'd prefer to remove this part.

Comment (samajammin) [reply src/content/bridges/index.md:42]: Agree, not sure the reasoning behind mentioning this particular company. Smells like link-building 😑

**Comment by shelleyolivia** [comment]
Hey @samajammin - yes I work at Alchemy. I focus on developer education and community! Big fan, love what you guys are doing at Ethereum Foundation. My goal with these couple of submissions is to add useful content to make the articles more educational and clear. Pls feel free to share any additional feedback on how I can make my PRs better for the community :) Appreciate you taking the time to review, and appreciate what you guys do for the web3 ecosystem! Thanks, Shelley

File: src/content/bridges/index.md (hunk 2 above)
Comment (minimalsm) [inline src/content/bridges/index.md:42]:
```suggestion
```

File: src/content/bridges/index.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ You're from the USA and are planning a trip to Europe. You have USD, but you nee
 
 But, what do you do if you want to make a similar exchange to use a different blockchain? Let's say you want to exchange ETH on Ethereum Mainnet for ETH on [Arbitrum](https://arbitrum.io/). Like the currency exchange we made for euros, we need a mechanism to move our ETH from Ethereum to Arbitrum. Bridges make such a transaction possible. In this case, [Arbitrum has a native bridge](https://bridge.arbitrum.io/) that can transfer ETH from Mainnet onto Arbitrum.
 
+
```
Comment (minimalsm) [inline src/content/bridges/index.md:24]:
```suggestion
```

**Review by minimalsm — APPROVED** [review]
Thanks @shelleyolivia, I've removed the section I commented on. Merging 🚀

**Pull request merged**